src/lib/funding.js

```diff
@@ -113,3 +116,28 @@ export function precacheRememberedFunding() : ZalgoPromise<void> {
     }
     return ZalgoPromise.resolve();
 }
+
+export function getRenderedButtons(props : Object) : string {
+    const {
+        commit,
+        env,
+        funding,
+        locale: rawLocale,
+        style: {
+            layout,
+            label
+        }
+    } = props;
+
+    const locale = rawLocale ? parseLocale(rawLocale) : getButtonConfig('DEFAULT', 'defaultLocale');
+    const selected = labelToFunding(label);
+    const sources = determineEligibleFunding({
+        funding,
+        selected,
+        locale,
+        env,
+        layout,
+        commit
+    });
+    return String(sources);
```

Comment:
Let's return an array. Zoid is smart enough to serialize that correctly for the query param.

Reply:
Done!